aquasec-enforcer/values.yaml

```diff
@@ -0,0 +1,33 @@
+imageCredentials:
+  repositoryUriPrefix: "docker.io"
+  registry: "index.docker.io/v1/" #for docker.io #REQUIRED
+  username: #REQUIRED
+  password: #REQUIRED
+  email:
+
+token: #REQUIRED
+
+server:
+  host: server-gateway
+  port: 3622
+
+rbac:
+  enabled: true
+  roleRef:
+
+image:
+  repository: aquasec/agent
+  tag: 3.2
+  pullPolicy: IfNotPresent
+
+# TODO: Fill-in the correct liveness and readiness probes
```

Comment:
@oranmoshai please provide the needed probes

Reply:
Currently we do not have healthcheck endpoint in the enforcer we will add it to the roadmap